scripts/releaseChangelog.mjs

```diff
@@ -264,6 +287,27 @@ async function main(argv) {
     year: 'numeric',
   });
 
+  const logCommunitySection = () => {
+    // TODO: separate first timers and regular contributors
+    const contributors = [
+      ...Array.from(community.contributors),
+      ...Array.from(community.firstTimers),
+    ].sort((a, b) => a.toLowerCase().localeCompare(b.toLowerCase()));
+    if (contributors.length === 0) {
+      return '';
+    }
+
+    return `Special thanks go out to our community contributors who have helped make this release possible:\n${contributors.join(', ')}.`;
+  };
+
+  const logTeamSection = () => {
+    return `Following are all team members who have contributed to this release:\n${Array.from(
```

Comment:
Are you sure we need this? 🤔 

By the way, WDYT about moving this to the top and replacing this existing somewhat pointless sentence:
```md
We'd like to offer a big thanks to the ${authors.length} contributors who made this release possible. Here are some highlights ✨
```
?

Reply:
> Are you sure we need this? 🤔

I am trying to align this with the approach the core team is taking and since they are currently listing all contributors (but will apply this change to their repo as well) they want to keep all mentions. Right @mnajdova?

> By the way, WDYT about moving this to the top and replacing this existing somewhat pointless sentence:

I thought about this as well. Might be worth a shot. We can always move it back if we think this looks odd or misplaced. 👍🏼